code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -297,7 +297,12 @@
 		cooldown = COOLDOWN_DAMAGE
 		for(var/V in listeners)
 			var/mob/living/L = V
-			L.apply_damage(15 * power_multiplier, def_zone = BODY_ZONE_CHEST, wound_bonus=CANT_WOUND)
+			if(ismegafauna(L) || istype(L, /mob/living/simple_animal/hostile/asteroid))
+				L.apply_damage(360 * power_multiplier, def_zone = BODY_ZONE_CHEST, wound_bonus=CANT_WOUND) // very long cooldown so make it really good at killing lavaland mobs
+			else if(L.mind?.martial_art && L.mind.martial_art.id == "ultra violence")
```

Comment:
Why does Ultra Violence (If im reading this right) get a buff from this? I'm not sure how the two connect honestly.

Reply:
Not a buff to ultra violence, makes voice of god do more damage against IPCs that have it.

Reply:
Ah that makes sense now, I misread the line cause im an idiot. 

I still wonder why Ultra Violence is tied to this though? Is it lore-related?

Reply:
It's an ultrakill reference